lib/Common/Memory/RecyclerTelemetryInfo.h

```diff
@@ -85,6 +85,28 @@ namespace Memory
 #endif
     };
 
+    /**
+     * Consolidated summary of data from RecyclerFlagsTable that we want to
+     * transmit via telemetry. Goal is to pack this into maximum of 64-bits.
+     */
+    struct RecyclerFlagsTableSummary
+    {
+        union
```

Comment:
Unless there's something I'm missing about MSVC compiler guarantees and telemetry infrastructure, reading from a member of a union that was written to by multiple names is only guaranteed to yield what you put in if you read from the most-recently-written name (everything else is undefined behavior).

Reply:
This is a bad definition.... :) fixing...

Reply:
thanks, this should be fixed now.